core/common/src/main/java/alluxio/grpc/GrpcExceptionUtils.java

```diff
@@ -27,156 +22,13 @@
  */
 @ThreadSafe
 public final class GrpcExceptionUtils {
-  private static final Logger LOG = LoggerFactory.getLogger(GrpcExceptionUtils.class);
-
-  private static final String sInnerCauseKeyName = "grpc-exception-cause-bin";
-  private static final Metadata.Key<byte[]> sInnerCauseKey =
-      Metadata.Key.of(sInnerCauseKeyName, Metadata.BINARY_BYTE_MARSHALLER);
-
-  private GrpcExceptionUtils() {} // prevent instantiation
-
-  /**
-   * Convert from grpc exception to alluxio status exception.
-   *
-   * @param e the grpc exception
-   * @return the alluxio status exception
-   */
-  public static AlluxioStatusException fromGrpcStatusException(StatusRuntimeException e) {
-    alluxio.exception.status.Status alluxioStatus;
-    switch (Status.fromThrowable(e).getCode()) {
```

Comment:
@ggezer is `Status.fromThrowable(e)` different from `e.getStatus()`?

Reply:
It shoudn't be. However I think you should be prepared to get null from e.getStatus() unless it's guaranteed to return a valid value.

Reply:
I looked into `StatusRuntimeException`, and they guarantee that `getStatus()` returns non-null. If you try to construct a `StatusRuntimeException` with a `null` status, it causes an NPE